src/tests/unit/pizzas-dao.test.js

```diff
@@ -0,0 +1,156 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import { getPizzaByIdData } from './test-data';
+
+const should = chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+let connectionStub;
+let getDoughByIdStub;
+let pizzasDao;
+let result;
+let resultError;
+let serializePizzaStub;
+
+const {
+  baseSerializerReturn,
+  emptyDbReturn,
+  fullRawPizza,
+  getPizzaByIdQuery,
+  getPizzaIngredientsQuery,
+  multipleRowsReturn,
+  pizzaDbReturn,
+  pizzaIngredientsDbReturn,
+  serializedDoughReturn,
+} = getPizzaByIdData;
+
+const setUpDatabaseStubs = (firstCall, secondCall, serializerReturn) => {
+  connectionStub = sinon.stub();
+  connectionStub.onCall(0).resolves(firstCall);
+  connectionStub.onCall(1).resolves(secondCall);
+  getDoughByIdStub = sinon.stub().returns(serializerReturn);
+};
+
+const proxyquirePizzasDao = () => {
+  const getConnectionStub = sinon.stub().resolves({
+    execute: connectionStub,
+    close: () => {},
+  });
+
+  return proxyquire('api/v1/db/oracledb/pizzas-dao', {
+    './connection': {
+      getConnection: getConnectionStub,
+    },
+    '../../serializers/pizzas-serializer': {
+      serializePizza: serializePizzaStub,
+    },
+    './doughs-dao': {
+      getDoughById: getDoughByIdStub,
+    },
+  });
+};
+
+
+describe('test pizzas DAO', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+  });
+  after(() => {
+    sinon.restore();
+  });
```

Comment:
This can be simplified to `after(() => sinon.restore());`

Reply:
The `before` can also be simplified that way but it might make the line to long. I can't tell with my eyes.

Reply:
looks like both are good to go 👍

Reply:
Should this be `beforeEach` or just `before`? https://mochajs.org/#hooks

Reply:
just `after`, this is to reset the configuration stub. The lower nested `context` blocks reset their sinon stubs individually.